### DISCUSSION TITLE
problème ouverture d'un fichier image

### DISCUSSION THREAD

**Discussion by gghyvelde** [arkypita/LaserGRBL#1493] [Q&A]
**Describe the bug**

Windows 10

LaserGrbl v4.3.0.0
Grbl v1.1f
Wrapper: UsbSerial
Win 10, Home, 10.0.21354.0 (7)
CLR: C:\Windows\Microsoft.NET\Framework64\v4.0.30319\mscorlib.dll 4.8.4341.0 built by: NET48REL1LAST_C

TypeOf exception  [System.OverflowException]
Exception message [La valeur était trop grande ou trop petite pour un Int32.]
Exception source  [mscorlib], thread [Main Thread]
Exception method  [Int32 ToInt32(System.Decimal)]

   ----------- stack trace -----------
   à System.Decimal.ToInt32(Decimal d)
   à LaserGRBL.RasterConverter.ConvertSizeAndOptionForm.AssignMinMaxLimit() dans E:\LaserGRBL\LaserGRBL\RasterConverter\ConvertSizeAndOptionForm.cs:ligne 89
   à LaserGRBL.RasterConverter.ConvertSizeAndOptionForm..ctor(GrblCore core) dans E:\LaserGRBL\LaserGRBL\RasterConverter\ConvertSizeAndOptionForm.cs:ligne 47
   à LaserGRBL.RasterConverter.RasterToLaserForm.BtnCreateClick(Object sender, EventArgs e) dans E:\LaserGRBL\LaserGRBL\RasterConverter\RasterToLaserForm.cs:ligne 175
   à System.Windows.Forms.Control.OnClick(EventArgs e)
   à System.Windows.Forms.Button.OnClick(EventArgs e)
   à System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   à System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   à System.Windows.Forms.Control.WndProc(Message& m)
   à System.Windows.Forms.ButtonBase.WndProc(Message& m)
   à System.Windows.Forms.Button.WndProc(Message& m)
   à System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

**Comment by arkypita** [comment]
I believe that your GRBL configuration is corrupted.
Check your $$ config, specially table size configuration.
